examples/counter.js

```diff
@@ -12,7 +12,7 @@ engine.addListener((store, actions, actionName, actionArguments) => {
   console.log(actionName, actionArguments, '->', store)
 })
 
-engine.actions.add(5) // add 5 -> { num: 5 }
+engine.actions.add(5, 20) // add 5 -> { num: 5 }
```

Comment:
Is this adding 5 plus 20? If so, why is the num 5?

Reply:
good catch, this was for testing, will remove